dashboard/config/libraries/PoemBot.interpreted.js

```diff
@@ -53,19 +123,94 @@ function randomWord() {
 }
 
 function addLine(line) {
-  poem.lines.push(line || '');
+  poem.lines.push(line || [blankText]);
 }
 
-function setTitle(title) {
-  poem.title = title;
+function setTitle(line) {
+  if (line) {
+    poem.title = line[0].value;
+  }
 }
 
-function setAuthor(author) {
-  poem.author = author;
+function setAuthor(line) {
```

Comment:
why does this block accept an array now?

Reply:
That's just to make the types work out, because you could put a concatenated text block in the socket. For now I just have it using the first string value but we'll probably want to flatten? Or maybe make this a field instead of a socket input.